src/operators.jl

```diff
@@ -440,64 +442,36 @@ function _multiplyt!(ret::Array{T}, lhs::Matrix, rhs::SparseMatrixCSC) where T<:
 end
 
 
-# TODO: implement sparse * sparse code as in base/sparse/linalg.jl (spmatmul)
-_multiply!(ret::AbstractArray{T}, lhs::SparseMatrixCSC, rhs::SparseMatrixCSC) where {T<:JuMPTypes} = _multiply!(ret, lhs, full(rhs))
-_multiplyt!(ret::AbstractArray{T}, lhs::SparseMatrixCSC, rhs::SparseMatrixCSC) where {T<:JuMPTypes} = _multiplyt!(ret, lhs, full(rhs))
+function Base.Matrix(S::SparseMatrixCSC{V}) where V<:Variable
```

Comment:
It is needed because of this bug right: https://github.com/JuliaLang/julia/issues/27015 ?

Reply:
I believe that's one of the reasons.  Also remember there's no `dense` in 1.0, so you need a safe way of converting to a dense matrix.

Reply:
It was added in master to fix https://github.com/JuliaOpt/JuMP.jl/issues/1275. and there is a comment `See https://github.com/JuliaLang/julia/issues/27015` explaining why it is needed on master above the method. I think we should add it here too